packages/beachball/src/commands/sync.ts

```diff
@@ -14,17 +14,33 @@ export async function sync(options: BeachballOptions) {
       continue;
     }
 
-    const npmArgs = ['view', pkg, 'version'];
+    if (semver.prerelease(info.version)) {
+      console.log(`Warning: cannot sync prerelease package "${pkg}@${info.version}"`);
```

Comment:
can you elaborate why we cannot sync prerelease?

Reply:
Because as far as I'm aware you can't tell in an automated way which prerelease version comes after the current one.

Reply:
Also the right thing to do in that situation is unclear: do they want the next prerelease version (which again we can't determine), or the next non-prerelease?

Reply:
i'm confused. 
1. are you saying we cannot compare which version is newer on this line:
```
semver.lt(info.version, publishedVersion)
```
2. sync is to sync to the published version. so whatever version it's currently published in npm, we update local version to that if local version is older

Reply:
Thinking about this some more and came to this conclusion: the sync is doing too much here. When I designed this command, the idea is to have it sync down whatever is the @latest dist-tag from npm registry regardless of whatever format it is. This command is really meant as a way to reset your repo to whatever state is captured inside a npm registry. We should actually REMOVE logic regarding `semver.lt()` below.